batch7crud-roll3/src/main/java/com/lftechnology/batch7crud/controller/CustomHttpServlet.java

```diff
@@ -22,28 +26,6 @@
 public abstract class CustomHttpServlet extends HttpServlet {
   private static final Logger LOGGER = Logger.getLogger(CustomHttpServlet.class.getName());
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "LOGGER" private field. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1068)